drivers/video/mt9m114.c

```diff
@@ -381,6 +387,26 @@ static int mt9m114_set_output_format(const struct device *dev, int pixel_format)
 	return ret;
 }
 
+static int mt9m114_set_orientation(const struct device *dev, int enable, uint8_t mask)
```

Comment:
I would drop this function. Just use `mt9m114_modify_reg()` directly as suggested in comments below

Reply:
Now that `mt9m114_modify_reg()` is used everywhere, this function can be removed.
Thanks for the modifications!